tests/commands/create/test_install_image.py

```diff
@@ -51,7 +51,7 @@ def test_no_requested_size(create_command, tmp_path, capsys):
     )
 
     # The right message was written to output
-    expected = "\nInstalling input/original.png as sample image... done\n"
+    expected = "Installing input/original.png as sample image... done\n\n"
     assert capsys.readouterr().out == expected
```

Comment:
Sooo....I don't really understand this change. When I use the Wait Bar, I don't see line breaks like this or any changes from how it worked in the past. My best guess is something subtle has changed about the way the dynamism of the Wait Bar is interacting with `capsys`...

Reply:
I can't spot any differences in "real world" output either; capsys is doing enough weirdness that I'm happy to live with this as an explanation for now.